### ISSUE TITLE
Lazer > Stable > Lazer map timing export bug

### ISSUE THREAD

**Issue by shrublet** [ppy/osu#23826]
### Type

Game behaviour

### Bug description

After creating a map in Lazer, exporting it, modifying it in stable, and re-importing it into Lazer, it caused odd SV issues related to timing points. In stable, the only things I did was add timing points to mute slider ends and fix incorrectly imported sliders (lazer sliders that used new node types).

### Screenshots or videos

This is what the map looks like in Lazer after exporting to stable, adding timing points, exporting from stable, and re-importing into Lazer.

https://github.com/ppy/osu/assets/63182228/34648c24-7720-49d0-bc17-142d20859d2b

This is what the edited diff looks like in stable. AI mod only brings up other issues (no hitsounds, mp3 too long, stack leniency, countdown conflict, and spread needs easier diff).

https://github.com/ppy/osu/assets/63182228/68ccfc3b-c3d7-44b9-bbd0-74557da53a13

I've also attached a zip with both difficulties and the audio/bg.

[Tsukumo Sana - Astrogirl (Shrub).zip](https://github.com/ppy/osu/files/11699459/Tsukumo.Sana.-.Astrogirl.Shrub.zip)

### Version

2023.605.0-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/11699471/database.log)
[input.log](https://github.com/ppy/osu/files/11699472/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11699473/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/11699474/network.log)
[performance.log](https://github.com/ppy/osu/files/11699475/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/11699476/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/11699477/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/11699478/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/11699479/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/11699480/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11699481/updater.log)

**Comment by Dimmitsaras** [comment]
Related https://github.com/ppy/osu/issues/17258

**Comment by peppy** [comment]
Bumping to p1 until investigated.

**Comment by frenzibyte** [comment]
It's quite hard to follow where the bug specifically is since the original beatmap isn't attached, but I believe this should be at least mitigated with the recently added "export for compatibility" function (see https://github.com/ppy/osu/pull/24186).

@shrublet check whether the above does indeed fix your issue, and report back if it doesn't.
